packages/relay/src/RegistrationManager.ts

```diff
@@ -397,7 +397,13 @@ export class RegistrationManager {
       transactions = transactions.concat(txHash)
       skipGasEstimationForRegisterRelay = true
     }
-    const registerMethod = await this.contractInteractor.getRegisterRelayMethod(this.hubAddress, this.config.url)
+    let registeredUrl: string
+    if (this.config.privateMode) {
+      registeredUrl = ''
```

Comment:
instead of ignoring the config url, maybe private mode should validate the url is not set

Reply:
Well, I was thinking a relay operator who wants to change from private to public would rather flip a flag than remove/add the string. Especially relevant for testing.